recipe/conda_build_config.yaml

```diff
@@ -424,7 +424,9 @@ giflib:
 glew:
   - 2.1
 glib:
-  - 2.58
+  - 2.58  # [not (osx and arm64)]
+  # Can't update to 2.66 on `osx and x86_64` yet because that requires a newer macOS version than 10.9
+  - 2.66  # [osx and arm64]
```

Comment:
There was talk of changing this to `2` after the migration was done?

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1027#issuecomment-748063425

Reply:
Oops this review was meant for https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1137/